src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(res, number) {
+      return res + number;
+    },
+
+    subtract(res, number) {
+      return res - number;
+    },
+
+    multiply(res, number) {
+      return res * number;
+    },
+
+    divide(res, number) {
+      return res / number;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate(operation, number) {
+      this.result = operation(this.result, number);
+
+      return this;
```

Comment:
```suggestion
  operate(callback, value) {
      this.result = callback(value);

      return this;
    },
```

Reply:
use proper names for variables - if you want to name variable result, pick a better one :)